app/models/user.rb

```diff
@@ -45,7 +45,7 @@ def self.change_code(type, newCode)
   end
 
   ##Methods dealing with search
-  @@member = ["first", "last", "team", "major", "skillset", "linkedinLstring", "facebook", "year"]
+  @@member = ["first", "last", "team", "major", "skillset", "year"]
   @@admin_only = ["email", "sid", "code"]
```

Comment:
I believe Team-leader? is a column now for admin? It should also be added to the @@admin_only array.